modules/dashboards/application/controllers/IndexController.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Icinga\Module\Dashboards\Controllers;
+
+use Icinga\Module\Dashboards\Common\Database;
+use Icinga\Module\Dashboards\Web\Controller;
+use Icinga\Module\Dashboards\Web\Widget\DashboardWidget;
+use Icinga\Web\Url;
+use ipl\Sql\Select;
+
+class IndexController extends Controller
+{
+    use Database;
+
+    public function indexAction()
+    {
+        $this->createTabs();
+
+        $select = (new Select())
+            ->columns('dashlet.name, dashlet.dashboard_id, dashlet.url')
+            ->from('dashlet')
+            ->join('dashboard d', 'dashlet.dashboard_id = d.id')
+            ->where(['d.id = ?' => $this->getTabs()->getActiveName()]);
+
+        $dashlets = $this->getDb()->select($select);
+
+        $this->content = new DashboardWidget($dashlets);
+    }
+
+    protected function createTabs()
```

Comment:
Document what this function does and what it returns. Maybe we should rename this function to `createTabsAndAutoActivateDashboard()` or something like that.